src/views/LoginView.vue

```diff
@@ -96,6 +162,7 @@ form {
   display: flex;
   flex-direction: column;
 }
+
```

Comment:
Tag even ergens hier een issue voor de hoeveelheid CSS.

Reply:
added in commit https://github.com/GEWIS/sudosos-frontend-vue3/pull/31/commits/c0cd5afae61160be4c481f76a38752281f930aa6